.github/workflows/smoke.yml

```
@@ -0,0 +1,126 @@
+          
+      - name: Install dependencies
+        run: npm ci
+        
+      - name: Run smoke tests
+        env:
+          BASE_URL: ${{ matrix.environment == 'staging' && 'https://staging.siraj.life' || 'https://siraj.life' }}
+          NODE_ENV: production
+        run: |
+          echo "🧪 Running smoke tests against: $BASE_URL"
+          npx tsx scripts/smoke-prod.ts > smoke-results-${{ matrix.environment }}.log 2>&1
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)